RNA vaccines section (#219)
greenelab/covid19-review: content/20.treatments.md

#### RNA Vaccines

RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response. 
The mRNA technology is transcribed _in vitro_ and delivered to cells via lipid nanoparticles (LNP). 

> They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594]. 

Reviewer: ```suggestion
They are recognized by ribosomes _in vivo_ and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594]. 
```

> The resulting intracellular viral proteins are displayed on surface MHC I proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi:10.3389/fimmu.2019.00594]. 

Reviewer: It's definitely correct that presentation on MHC I will generate a CD8 T cell response, but I think to get the CD4 T cell response (and subsequent Ab induction) you need signaling through the MHC II pathway as well. I suggest clarifying that here, or perhaps just saying "MHC proteins" instead of "MHC I proteins" here?

Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm. 
The LNP covering protects the mRNA from enzymatic degradation outside of the cell [@doi:10.4155/tde-2016-0006]. 
Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells. 
Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].

mRNA vaccines confer many advantages over traditional viral vectored vaccines and DNA vaccines. 

> In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, and thereby reduce the risk of a detrimental immune response in the host[@doi:10.1016/j.immuni.2020.03.007]. 

Reviewer 2: ```suggestion
In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, and thereby reducing the risk of a detrimental immune response in the host [@doi:10.1016/j.immuni.2020.03.007]. 
```

> Unlike DNA vaccines, mRNA technologies are naturally degradable, non-integrating, and do not need to cross the nuclear membrane in addition to the plasma membrane to see their effects [@doi:10.3389/fimmu.2019.00594].  

Reviewer 2: ```suggestion
Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen [@doi:10.3389/fimmu.2019.00594].  
```

> Furthermore, mRNA viruses are easily, affordably, and rapidly scalable.

Reviewer: ```suggestion
Furthermore, mRNA vaccines are easily, affordably, and rapidly scalable.
```

Author: thanks for catching this!

> Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have been licensed or commercialized thus far. Nevertheless, they have show promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, rabies, and cytomegalovirus [@url:https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 

Reviewer: This looks great! It's just a formatting thing, but I think because these are two separate sentences they should be on two separate lines?

> Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have been licensed or commercialized thus far. Nevertheless, they have show promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, rabies, and cytomegalovirus [@url:https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 

Reviewer 3: ```suggestion
Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have been licensed or commercialized thus far.
Nevertheless, they have show promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, rabies, and cytomegalovirus [@url:https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 
```

> mRNA vaccines have not been licensed or commercialized; however, several agents for cancer and infectious diseases, including rabies, coronavirus, influenza, rabies, and cytomegalovirus, have shown promising immunization in animal models and are currently in clinical trials [@url:https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 

Reviewer: This might flow a bit better if these two sentences were reworded a bit (just a suggestion, feel free to ignore if you disagree). Perhaps something like:

Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have been licensed or commercialized thus far. Nevertheless, they have show promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, rabies, and cytomegalovirus.

Author: Thank you this reads much nicer!

> Preclinical data from Pardi et al. identified effective antibody generation against full length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi:10.1038/s41467-018-05482-0]. 

Reviewer 2: ```suggestion
Preclinical data from Pardi et al. identified effective antibody generation against full-length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi:10.1038/s41467-018-05482-0]. 
```

> Similar immunological responses for mRNA vaccines were observed in humans in Curevac and Moderna operated Phase I and II clinical trials for rabies, flu, and zika [@doi:10.1080/14760584.2017.1355245]. 

Reviewer 2: ```suggestion
Similar immunological responses for mRNA vaccines were observed in humans in Phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika [@doi:10.1080/14760584.2017.1355245]. 
```

##### Anticipated Mechanism

Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm [@doi:10.4155/tde-2016-0006], and facilitate endosomal escape. 

> LNPs can be coated with modalities recognized and engulfed by specific celltypes. 

Reviewer: This is definitely a big advantage of using LNPs! Perhaps it might be good to give an example here to show what you mean?

Reviewer: (Also I think a space is missing between "cell types")

> There are three types of RNA vaccines: non-replicating, in vivo self-replicating, and in vitro dendritic cell non-replicating [@url:https://www.phgfoundation.org/briefing/rna-vaccines].  

Reviewer: ```suggestion
There are three types of RNA vaccines: non-replicating, _in vivo_ self-replicating, and _in vitro_ dendritic cell non-replicating [@url:https://www.phgfoundation.org/briefing/rna-vaccines].  
```

Non-replicating mRNA vaccines consist of a simple open reading frame (ORF) for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail. 

> In vivo self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245]. 

Reviewer: ```suggestion
_In vivo_ self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245]. 
```

> The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase. Structural proteins required for viral assembly are not included in the engineered genome [@doi:10.3389/fimmu.2019.00594]. 

Reviewer 3: ```suggestion
The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase.
Structural proteins required for viral assembly are not included in the engineered genome [@doi:10.3389/fimmu.2019.00594]. 
```

Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response[@url:https://www.phgfoundation.org/briefing/rna-vaccines].  

> Finally, in vitro dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells. 

Reviewer: ```suggestion
Finally, _in vitro_ dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells. 
```

Finally, in vitro dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells. 

> Dendritic cells are potent antigen presenting immune cells that easily take up mRNA and present it to other cells, including B-cells which generate antibodies against the protein. 

Reviewer: This sentence makes it sound as though DCs present mRNA, but in actuality I think they present peptide fragments. As you mentioned above, after the DCs are transfected ex vivo with the mRNA, their endogenous biosynthetic machinery should translate it into protein. The protein is then cleaved into peptide fragments that are presented on MHCI or MHCII. 

Also, I would suggest rewording the second clause in this sentence. DCs present antigen to T cells, not B cells. B cells can sample the antigen on their own, and are activated by interacting with T cells (ideally the T cells will have been activated by interacting with the DCs).

Antigen presentation in general is quite confusing! Let me know if anything doesn't make sense! :)

Author: I updated this section to be a little bit more specific without getting into too many details. How does it look now?

Reviewer: It looks perfect!!!! :)

> These cells are isolated from the patient, grown and transfected ex vivo, and re-introduced to the patient [@doi:10.1038/nrd.2017.243].

Reviewer: ```suggestion
These cells are isolated from the patient, grown and transfected _ex vivo_, and re-introduced to the patient [@doi:10.1038/nrd.2017.243].
```

#### RNA Vaccines

RNA vaccines are nucleic-acid based modalities that code for viral antigens against which our body elicits a humoral and cellular immune response. 
The mRNA technology is transcribed in vitro and delivered to cells via lipid nanoparticles (LNP). 

> They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@https://doi.org/10.3389/fimmu.2019.00594]. 

Reviewer 3: ```suggestion
They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594]. 
```
note the `@doi:` syntax that you used below works while this one using the full URL (`@https:`) will not.

> The resulting intracellular viral proteins are displayed on surface MHC I proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi: 10.3389/fimmu.2019.00594]. 

Reviewer 3: ```suggestion
The resulting intracellular viral proteins are displayed on surface MHC I proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi:10.3389/fimmu.2019.00594]. 
```

Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm. 

> The LNP covering protects the mRNA from enzymatic degradation outside of the cell [@doi: 10.4155/tde-2016-0006]. 

Reviewer 3: ```suggestion
The LNP covering protects the mRNA from enzymatic degradation outside of the cell [@doi:10.4155/tde-2016-0006]. 
```
This one is almost perfect! Just no space between `@doi:` and the identifier!

Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells. 

> Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell [@https://doi.org/10.3389/fimmu.2019.00594; @https://doi.org/10.1080/14760584.2017.1355245].

Reviewer 3: ```suggestion
Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].
```

mRNA vaccines confer many advantages over traditional viral vectored vaccines and DNA vaccines. 

> In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, and thereby reduce the risk of a detrimental immune response in the host[@doi: 10.1016/j.immuni.2020.03.007]. 

Reviewer 3: ```suggestion
In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, and thereby reduce the risk of a detrimental immune response in the host[@doi:10.1016/j.immuni.2020.03.007]. 
```

> Unlike DNA vaccines, mRNA technologies are naturally degradable, non-integrating, and do not need to cross the nuclear membrane in addition to the plasma membrane to see their effects [@https://doi.org/10.3389/fimmu.2019.00594].  

Reviewer 3: ```suggestion
Unlike DNA vaccines, mRNA technologies are naturally degradable, non-integrating, and do not need to cross the nuclear membrane in addition to the plasma membrane to see their effects [@doi:10.3389/fimmu.2019.00594].  
```

Furthermore, mRNA viruses are easily, affordably, and rapidly scalable

mRNA vaccines have been developed for therapeutic and prophylactic purposes. 

> mRNA vaccines have not been licensed or commercialized, however, several agents for cancer and infectious diseases, including rabies, coronavirus, influenza, rabies, and CMV, have shown promising immunization in animal models and are currently in clinical trials [@url: https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 

Author: Yeah CMV= cytomegalovirus! Thanks for catching the abbreviation!

> Preclinical data from Pardi et al. identified effective antibody generation against full length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi: https://doi.org/10.1038/s41467-018-05482-0]. 

Reviewer 3: ```suggestion
Preclinical data from Pardi et al. identified effective antibody generation against full length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi:10.1038/s41467-018-05482-0]. 
```

> Similar immunological responses for mRNA vaccines were observed in humans in Curevac and Moderna operated Phase I and II clinical trials for rabies, flu, and zika [@https://doi.org/10.1080/14760584.2017.1355245]. 

Reviewer 3: ```suggestion
Similar immunological responses for mRNA vaccines were observed in humans in Curevac and Moderna operated Phase I and II clinical trials for rabies, flu, and zika [@doi:10.1080/14760584.2017.1355245]. 
```

> Similar immunological responses for mRNA vaccines were observed in humans in Curevac and Moderna operated Phase I and II clinical trials for rabies, flu, and zika [@https://doi.org/10.1080/14760584.2017.1355245]. 

Author: This rephrasing sounds much better, thanks! However, I am not able to commit this suggestion because "outdated suggestions cannot be applied". Do you know how I could get around this?

##### Anticipated Mechanism

> Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm [@doi: 10.4155/tde-2016-0006], and facilitate endosomal escape. 

Reviewer 3: ```suggestion
Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm [@doi:10.4155/tde-2016-0006], and facilitate endosomal escape. 
```

LNPs can be coated with modalities recognized and engulfed by specific celltypes. 
LNPs 150nm or less effectively enter into lymphatic vessels. 

> There are three types of RNA vaccines: non-replicating, in vivo self-replicating, and in vitro dendritic cell non-replicating [@url: https://www.phgfoundation.org/briefing/rna-vaccines].  

Reviewer 3: ```suggestion
There are three types of RNA vaccines: non-replicating, in vivo self-replicating, and in vitro dendritic cell non-replicating [@url:https://www.phgfoundation.org/briefing/rna-vaccines].  
```
Very nice use of a URL citation! Just note that, like with doi, there's no space between the colon and the identifier.

Author: Thank you so much for catching the incorrect citation formats in this section!

Non-replicating mRNA vaccines consist of a simple open reading frame (ORF) for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail. 

> In vivo self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses [@https://doi.org/10.3389/fimmu.2019.00594; @https://doi.org/10.1080/14760584.2017.1355245]. 

Reviewer 3: ```suggestion
In vivo self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245]. 
```

> The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase. Structural proteins required for viral assembly are not included in the engineered genome [@https://doi.org/10.3389/fimmu.2019.00594]. 

Reviewer 3: ```suggestion
The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase. Structural proteins required for viral assembly are not included in the engineered genome [@doi:10.3389/fimmu.2019.00594]. 
```

> Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response[@url: https://www.phgfoundation.org/briefing/rna-vaccines].  

Reviewer 3: ```suggestion
Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response[@url:https://www.phgfoundation.org/briefing/rna-vaccines].  
```

Finally, in vitro dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells. 
Dendritic cells are potent antigen presenting immune cells that easily take up mRNA and present it to other cells, including B-cells which generate antibodies against the protein. 

> These cells are isolated from the patient, grown and transfected ex vivo, and re-introduced to the patient [@url:https://doi.org/10.1038/nrd.2017.243].

Reviewer 3: ```suggestion
These cells are isolated from the patient, grown and transfected ex vivo, and re-introduced to the patient [@doi:10.1038/nrd.2017.243].
```

##### Current Evidence

mRNA-1273 is the first COVID-19 vaccine to enter a phase I clinical in the United States.  

> ModernaTX, Inc. is currently spearheading an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-Cov-2 [@url: https://clinicaltrials.gov/ct2/show/study/NCT04283461]. 

Reviewer 3: ```suggestion
ModernaTX, Inc. is currently spearheading an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-Cov-2 [@url:https://clinicaltrials.gov/ct2/show/study/NCT04283461]. 
```

Forty-five participants will be enrolled in the study and given an intramuscular injection of mRNA-1273 in their deltoid muscle on Day 1 and Day 29, and then followed for the next twelve months. 
Healthy males and non-pregnant females aged 18-55 years are being recruited and will be divided in three dosage groups receiving either 25 micrograms, 100 mcg, or 250 mcg of the vaccine. 

The study started on March 3rd, 2020, and is expected to complete by June 1st, 2021. 
Emory Vaccine Center and Kaiser Permanent Washington Health Research Institute are currently recruiting participants with NIH Clinical Center expecting to recruit soon. 
Reports on patient safety and reactogenicity will be recorded soon. 

> IgG ELISA assays on patient serology samples will study the immunogenicity of the vaccine [@url: https://clinicaltrials.gov/ct2/show/study/NCT04283461].

Reviewer 3: ```suggestion
IgG ELISA assays on patient serology samples will study the immunogenicity of the vaccine [@url:https://clinicaltrials.gov/ct2/show/study/NCT04283461].
```
